pulp_rpm/app/models/repository.py

```diff
@@ -105,13 +108,22 @@ def get_downloader(self, remote_artifact=None, url=None, **kwargs):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+        permissions = [
+            ("create_remote_rpm", "Can create a remote"),
+            ("destroy_remote_rpm", "Can remove a remote"),
+            ("view_remote_rpm", "Can view a detail of remote"),
+            ("sync_remote_rpm", "Can start a sync"),
+            ("modify_remote_rpm", "Can modify content of a remote"),
+            ("manage_roles_remote_rpm", "Can manage roles on rpm remote"),
```

Comment:
create destroy and modify are autogenerated by django we don't need them explicitely.

Reply:
view as well